src/backend/resourcemanager/resqueuemanager.c

```diff
@@ -2142,8 +2142,10 @@ int registerConnectionByUserID(ConnectionTrack  conntrack,
 
 	transformConnectionTrackProgress(conntrack, CONN_PP_REGISTER_DONE);
 
-	elog(LOG, "ConnID %d. Connection is registered.", conntrack->ConnID);
-
+	elog(DEBUG3, "Resource queue %s has %d connections after registering "
+				 "connection.",
+				 queuetrack->QueueInfo->Name,
+				 queuetrack->CurConnCounter);
```

Comment:
It it better to add current ConnID here:
elog(DEBUG3, "Resource queue %s has %d connections after registering "
                                               "connection ConnID %d.",
                                               queuetrack->QueueInfo->Name,
                                               queuetrack->CurConnCounter,
                                               conntrack->ConnID);